HW03-VaughtK/hw03_VaughtK.py

```diff
@@ -0,0 +1,196 @@
+
+#################################################################################
+# Author: Kai Vaught
+# Username: VaughtK
+# Assignment: Homework 3
+# Purpose: To learn how branches work.
+# Google Doc Link: https://docs.google.com/document/d/1OvsmR456y61n3UzwBlIlVE13IBz0tF39Vyv10F24ZGI/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+import turtle
+
+t = turtle.Turtle()
+
+turtle2 = turtle.Turtle()
```

Comment:
These two turtles should be outside of `main()`. Move them as the first two lines of your `main()` function.